### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by jsoref**
This is mostly punctuation/style.

Technically I'm fixing the spelling of `macOS` (old spellings such as `OS X` and `Mac OS X` have been obsolete for a long time) -- this includes one comment in a Java file beyond the Readme, but it seemed better to include it here than split it out...

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -49,17 +49,17 @@ Before you start running the unit and integration tests in the Neo4j Maven proje
 
 == Running Neo4j ==
 
-After running a `mvn clean install` cd into `packaging/standalone/target` and extract the version you want, then
+After running a `mvn clean install`, **cd** into `packaging/standalone/target` and extract the version you want, then:
```
Comment (klaren) [inline README.asciidoc]: We use the code syntax for actual commands.
```suggestion
After running a `mvn clean install`, `cd` into `packaging/standalone/target` and extract the version you want, then:
```

File: README.asciidoc [hunk 2]
```diff
@@ -49,17 +49,17 @@ Before you start running the unit and integration tests in the Neo4j Maven proje
...
 
-in the extracted folder to start Neo4j on `localhost:7474`. On Windows you want to run
+in the extracted folder to start Neo4j on `localhost:7474`. On Windows you want to run:
 
   bin\neo4j start
 
 instead.
 
-Or, if you are not interested in the tarballs and just want to run Neo4j you can instead run a `mvn clean install -DminimalBuild` and then cd into `community/server` and run
+Or, if you are not interested in the tarballs and just want to run Neo4j you can instead run a `mvn clean install -DminimalBuild` and then **cd** into `community/server` and run:
```
Comment (klaren) [inline README.asciidoc]: Same here
```suggestion
Or, if you are not interested in the tarballs and just want to run Neo4j you can instead run a `mvn clean install -DminimalBuild` and then `cd` into `community/server` and run:
```

**Review by klaren — CHANGES_REQUESTED** [review]
Two small comments, otherwise this look good.

Thanks for the PR.

File: README.asciidoc (hunk 1 above)
Comment (jsoref) [reply README.asciidoc]: changes applied to the underlying commit. Thanks.

**Comment by jsoref** [comment]
@klaren: this should be ready

**Review by klaren — APPROVED** [review]

**Comment by klaren** [comment]
Thanks! You know the drill, build bot will pick it up later and merge it.

**Pull request closed without merging**